bindata/network/ovn-kubernetes/ovnkube-node.yaml

```diff
@@ -152,44 +145,27 @@ spec:
           name: host-cni-netd
         - mountPath: /var/lib/cni/networks/ovn-k8s-cni-overlay
           name: host-var-lib-cni-networks-ovn-kubernetes
+        - mountPath: /var/run/ovn-config
+          name: ovn-config-mount
+        - mountPath: /var/run/ovnkube-config
+          name: ovnkube-config-mount
 
         resources:
           requests:
             cpu: 100m
             memory: 300Mi
         env:
         - name: OVN_DAEMONSET_VERSION
-          value: "3"
-        - name: OVNKUBE_LOGLEVEL
-          value: "5"
-        - name: OVN_NET_CIDR
-          valueFrom:
-            configMapKeyRef:
-              name: ovn-config
-              key: net_cidr
-        - name: OVN_SVC_CIDR
-          valueFrom:
-            configMapKeyRef:
-              name: ovn-config
-              key: svc_cidr
+          value: "4"
         - name: K8S_NODE
           valueFrom:
             fieldRef:
               fieldPath: spec.nodeName
-        - name: K8S_APISERVER
-          valueFrom:
-            configMapKeyRef:
-              name: ovn-config
-              key: k8s_apiserver
-        - name: OVN_KUBERNETES_NAMESPACE
-          valueFrom:
-            fieldRef:
-              fieldPath: metadata.namespace
 
         lifecycle:
           preStop:
             exec:
-              command: ["rm","-f","/etc/cni/net.d/10-ovn-kubernetes.conf", "/host/opt/cni/bin/ovn-k8s-cni-overlay"]
+              command: ["/root/ovnkube.sh", "cleanup-ovn-node"]
```

Comment:
@pecameron same here, I think we should stick with what we currently have for the preStop hook. Otherwise we'll blow away a bunch of our logical network stuff and that'll disrupt networking during upgrades. We should just ignore the upstream cleanup-ovn-node stuff.

Reply:
OK